#! /bin/sh /usr/share/dpatch/dpatch-run
## 80_move_doc by  <piem@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: moving /u/s/kino/help to /u/s/doc/kino/help

@DPATCH@
--- a/src/commands.cc
+++ b/src/commands.cc
@@ -424,7 +424,7 @@
 void showHelp( const char *page )
 {
 	extern char* g_help_language;
-	std::string cmd = "\"" DATADIR "/kino/scripts/help.sh\" \"file://" DATADIR "/kino/help/";
+	std::string cmd = "\"" DATADIR "/kino/scripts/help.sh\" \"file://" DATADIR "/doc/kino/help/";
 	if ( page == NULL || strcmp( page, "" ) == 0 )
 		page = common->getCurrentPage()->getHelpPage().c_str();
 	cmd += std::string( g_help_language ) + std::string( "\" \"" ) + std::string( page ) + "\" &";
